src/components/AddingBooks.js

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
-import './Styles/Books.css';
+// import React from 'react';
+// import './Styles/Books.css';
 
-function AddBook() {
-  return (
-    <div className="Book-F">
-      <h3>Add New Book</h3>
-      <form>
-        <input
-          type="text"
-          placeholder="Enter Book Name"
-        />
-        <input
-          type="text"
-          placeholder="Enter Author Name"
-        />
-        <button className="Add-Book" type="submit"> Add Book </button>
-      </form>
-    </div>
-  );
-}
+// function addBook() {
+//   return (
+//     <div className="Book-F">
+//       <h3>Add New Book</h3>
+//       <form>
+//         <input
+//           type="text"
+//           placeholder="Enter Book Name"
+//         />
+//         <input
+//           type="text"
+//           placeholder="Enter Author Name"
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history. Please remove this and all other commented codes. Also, if the file itself is not used in the project, consider removing the file itself.

Reply:
@TadesseDev  Hi thank you for reviewing my project . don't you think that was to be an optional suggestion. not sure if it was necessary to mark as changes required..

Reply:
Dear @hetrox8, this is not the only reason why your project is in the required change state. Besides that according to microverse guidelines, any commented code is to be marked as required changes, This is to make sure students take this issues seriously.

Reply:
@TadesseDev wow sorry bruh i guess i forgot that over the modules.